manifests/p/pdftoppm/pdftoppm/24.08.0-0/pdftoppm.pdftoppm.installer.yaml

```
@@ -0,0 +1,40 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: pdftoppm.pdftoppm
+PackageVersion: 24.08.0-0
+InstallerType: zip
+NestedInstallerType: zip
+NestedInstallerFiles:
+- RelativeFilePath: poppler-24.08.0\Library\bin\pdfattach.exe
+  PortableCommandAlias: pdfattach
+- RelativeFilePath: poppler-24.08.0\Library\bin\pdfdetach.exe
+  PortableCommandAlias: pdfdetach
+- RelativeFilePath: poppler-24.08.0\Library\bin\pdffonts.exe
+  PortableCommandAlias: pdffonts
+- RelativeFilePath: poppler-24.08.0\Library\bin\pdfimages.exe
+  PortableCommandAlias: pdfimages
+- RelativeFilePath: poppler-24.08.0\Library\bin\pdfinfo.exe
+  PortableCommandAlias: pdfinfo
+- RelativeFilePath: poppler-24.08.0\Library\bin\pdfseparate.exe
+  PortableCommandAlias: pdfseparate
+- RelativeFilePath: poppler-24.08.0\Library\bin\pdftocairo.exe
+  PortableCommandAlias: pdftocairo
+- RelativeFilePath: poppler-24.08.0\Library\bin\pdftohtml.exe
+  PortableCommandAlias: pdftohtml
+- RelativeFilePath: poppler-24.08.0\Library\bin\pdftoppm.exe
+  PortableCommandAlias: pdftoppm
+- RelativeFilePath: poppler-24.08.0\Library\bin\pdftops.exe
+  PortableCommandAlias: pdftops
+- RelativeFilePath: poppler-24.08.0\Library\bin\pdftotext.exe
+  PortableCommandAlias: pdftotext
+- RelativeFilePath: poppler-24.08.0\Library\bin\pdfunite.exe
+  PortableCommandAlias: pdfunite
+- RelativeFilePath: poppler-24.08.0\Library\bin\zstd.exe
+  PortableCommandAlias: zstd
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

- https://github.com/microsoft/winget-pkgs/pull/232694#issuecomment-2679296948